packages/constants/index.js

```diff
@@ -35,4 +35,5 @@ export const GRAPHQL_TARGETS = {
   DASHBOARD_SERVICE: 'dashboard',
   PIM_INDEXER: 'pim-indexer',
   SETTINGS_SERVICE: 'settings',
+  ADMINISTRATION_SERVICE: 'core-ws',
```

Comment:
You mean `ADMINISTRATION_SERVICE: 'administration'` ?

Reply:
Et voila.